src/routes/mods/+page.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+	import Breadcrumbs from "$lib/components/Breadcrumbs.svelte";
+	import { page } from "$app/stores";
+	let showSignInButton = false;
+	import Signin from "$lib/components/Signin.svelte";
+	const mods = [
+	{
+	name: "Paper Mario: Master Quest",
+	description:
+	'Considered "The Dark Souls of Paper Mario", Master Quest is incredibly difficult. It was designed with elite challenge runners in mind, and unless you have extensive knowledge of Paper Mario, you may struggle with much of the content.',
+	link: "https://github.com/Brotenko/PMMasterQuest",
+	},
+	{
+	name: "Paper Mario Mod 2",
+	description: "This is a description of Mod 2.",
+	link: "#",
+	},
+	];
+
+	function signIn() {
+	window.location.href = "/auth/signin";
+	}
+	function signOut() {
+	window.location.href = "/auth/signout";
+	}
```

Comment:
Needs formatting, do `pnpm run format` and it should hopefully fix this
if not thats a bug 💀